scss/mixins/_grid.scss

```diff
@@ -10,6 +10,15 @@
   @include clearfix();
 }
 
+@mixin make-container-breakpoints($max-widths: $container-max-widths) {
```

Comment:
A more descriptive name would be `make-container-max-widths`. The comment should be by the mixin definition.